### PR TITLE
Partial Storybook CSF 3 story conversions

### PR DESCRIPTION
**Pull request by LFDanLu**
Closes 

## ✅ Pull Request Checklist:

- [ ] Included link to corresponding [React Spectrum GitHub Issue](https://github.com/adobe/react-spectrum/issues).
- [ ] Added/updated unit tests and storybook for this change (for new code or code which already has tests).
- [ ] Filled out test instructions.
- [ ] Updated documentation (if it already exists for this component).
- [ ] Looked at the Accessibility Practices for this feature - [Aria Practices](https://www.w3.org/TR/wai-aria-practices-1.1/)

## 🧢 Your Project:

RSP

### CODE REVIEW

File: packages/@react-spectrum/actionbar/stories/ActionBar.stories.tsx [hunk 1]
```diff
@@ -11,24 +11,43 @@
...
+  },
+  argTypes: {
+    onAction: {
+      table: {
+        disable: true
+      }
+    },
+    isEmphasized: {
+      control: 'boolean',
+      defaultValue: false
```
[the remark this answers is not in the document]
Comment (LFDanLu) [reply packages/@react-spectrum/actionbar/stories/ActionBar.stories.tsx:35]: Makes sense, I'll see about initializing them as null or making it so the controls are toggle on to apply first

[the remark this answers is not in the document]
Comment (snowystinger) [reply packages/@react-spectrum/actionbar/stories/ActionBar.stories.tsx:35]: does initializing them as null make them undefined?

[the remark this answers is not in the document]
Comment (LFDanLu) [reply packages/@react-spectrum/actionbar/stories/ActionBar.stories.tsx:35]: turns out if I omit the default value, the control isn't initialized until the tester clicks the "set boolean" button
image

[the remark this answers is not in the document]
Comment (snowystinger) [reply packages/@react-spectrum/actionbar/stories/ActionBar.stories.tsx:35]: i think that makes sense right?

[the remark this answers is not in the document]
Comment (LFDanLu) [reply packages/@react-spectrum/actionbar/stories/ActionBar.stories.tsx]: yep, that will be the pattern going forward, allows the story to initialize with most values as undefined until we modify them via controls

**Review by reidbarber — APPROVED** [review]
There are a few things I noticed we can improve, but they can definitely be handled in a follow-up:

For argTypes that we don't want to appear in the Controls panel (i.e. `children`, `onOpenChange`), we can add `control: false`.

File: packages/@react-spectrum/color/stories/ColorSlider.stories.tsx [hunk 2]
```diff
@@ -10,132 +10,173 @@
...
+export type ColorSliderStory = ComponentStoryObj<typeof ColorSlider>;
+
+export default {
+  title: 'ColorSlider',
+  component: ColorSlider,
+  args: {
+    onChange: action('onChange'),
+    onChangeEnd: action('onChangeEnd')
+  },
+  argTypes: {
```
[the remark this answers is not in the document]
Comment (LFDanLu) [reply packages/@react-spectrum/color/stories/ColorSlider.stories.tsx:33]: good catch, I'll update

**Review by snowystinger — APPROVED** [review]

**Review by reidbarber — APPROVED** [review]

**Pull request merged**